### PR TITLE
Add github CI action

### PR DESCRIPTION
**Pull request by ederag**
Created by PkgTemplate.

This is a first attempt (the CI is not launched for the fork).
If this works, windows and osx should be added.

Hope this will help to unlock the other PRs.

### CODE REVIEW

**Comment by ederag** [comment]
Kind ping, @pkofod maybe ?

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1.6'
+          - '1.7'
```
Comment (thomvet) [inline .github/workflows/CI.yml:22]: Suggest to change to '1', so that it will always run on the most current minor release.

Comment (ederag) [reply .github/workflows/CI.yml:22]: OK, thanks for the review.
1.8 is out, so I'd just add `1`, when someone with access enables the workflow :
> First-time contributors need a maintainer to approve running workflows

Comment (thomvet) [reply .github/workflows/CI.yml:22]: I would if I could! :)

**Comment by pkofod** [comment]
absolutely sorry for dragging my feet. I appreciate the help, but I'm fixing this to get it processed today. Again, sorry.

**Pull request closed without merging**